grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -283,6 +283,36 @@ class InvoiceService {
         return invoice
     }
 
+    def refreshInvoiceValues(Invoice invoice) {
```

Comment:
Let's call this refreshInvoice or refreshInvoiceItems instead of refreshInvoiceValues as the latter is somewhat ambiguous.